chain/client/src/stateless_validation/chunk_endorsement/tracker_v1.rs

```diff
@@ -209,7 +203,9 @@ impl ChunkEndorsementTrackerInner {
         let protocol_version = self.epoch_manager.get_epoch_protocol_version(&epoch_id)?;
         if !checked_feature!("stable", StatelessValidation, protocol_version) {
             // Return an empty array of chunk endorsements for older protocol versions.
-            return Ok(ChunkEndorsementsState::Endorsed(None, vec![]));
+            let mut endorsement_stats = ChunkEndorsementsState::default();
```

Comment:
this is used multiple times, please extract into a separate method, something like `ChunkEndorsementsState::stateless_validation_disabled()` or `ChunkEndorsementsState::old_protocol_version()`

Reply:
With a future PR, we'll get rid of tracker_v1 and we'll have only one instance of this pattern left.